consumer.go

```diff
@@ -815,10 +830,16 @@ func (bc *brokerConsumer) handleResponses() {
 	for child := range bc.subscriptions {
 		result := child.responseResult
 		child.responseResult = nil
-
 		switch result {
 		case nil:
-			// no-op
+			if !child.replicaInited {
+				return
+			}
+			if bc.broker.ID() != child.preferredReadReplica {
```

Comment:
I think this would also do the same:
```go
if child.replicaInited && bc.broker.ID() != child.preferredReadReplica
```

not need to do the return there

also the `no-op` message might be important to leave it, maybe with a modified comment

Reply:
Ack. will do this